Detectors/GLOQC/include/GLOQC/MatchITSTPCQC.h

```diff
@@ -372,9 +385,18 @@ class MatchITSTPCQC
   float mEtaCut = 1e10f; // 1e10f as defaults of Detectors/GlobalTracking/include/GlobalTracking/TrackCuts.h
                          // TODO: define 2 different values for min and max (*)
 
-  ClassDefNV(MatchITSTPCQC, 2);
+  // for V0s
+  o2::vertexing::DCAFitterN<2> mFitterV0;
+  TH1F* mK0Mass = nullptr;
```

Comment:
just wondering if we should prefer `TH1D` (maybe I'm overthinking w/o a clear idea of the possible bin contents for this histogram at the QC async. phase, but with very large statistics e.g. in Pb-Pb we risk to loose sensitivity after the float mantissa precision pow(2,24)-1 = 16777215)

Reply:
Hello @mfaggin , 

I don't know, and I am actually a bit unsure. In the task there is a mix of TH1F and TH1D for efficiencies, e.g.:

```
  TH1D* mPtNum[matchType::SIZE] = {};
  TH1D* mPtDen[matchType::SIZE] = {};
```
but then:
```
  TH1F* mPhiNum[matchType::SIZE] = {};
  TH1F* mPhiDen[matchType::SIZE] = {};
```
In principle, these should have more entries than the K0s... 

I can change it if you prefer, the size will double.